open_spiel/games/quoridor/quoridor.cc

```diff
@@ -261,7 +281,7 @@ void QuoridorState::AddActions(Move cur, Offset offset,
   Move forward = cur + offset * 2;
   if (GetPlayer(forward) == kPlayerNone) {
     // Normal single step in this direction.
-    moves->push_back(forward.xy);
+    moves->push_back((base_for_relative_ + offset * 2).xy);
```

Comment:
why the * 2 on the offset here?

Is it to keep the coordinates even ? (otherwise they get interpreted as walls?)